extensions/wikia/VideoPageTool/templates/VideoPageAdminSpecial_featured.php

```diff
@@ -61,9 +61,9 @@
 				<textarea
 					maxlength="<?= $descriptionMaxLength ?>"
 					class="description"
-					id="description-<?= Sanitizer::encodeAttribute( $x ); ?>"
+					id="description-<?= $x ?>"
 					placeholder="<?= wfMessage( 'videopagetool-placeholder-video-description' )->escaped() ?>"
-					name="description[]"><?= htmlspecialchars( $video[ 'description' ] ); ?></textarea>
+					name="description[]"><?= $video[ 'description' ]; ?></textarea>
```

Comment:
Do you really want to revert changes introduced in a previous commit?

5f9e509ce94783e8907b9f33dd7a73c2499b4e2f